test/ui-test/testSuites/suite_status/tst_communityFlows/test.feature

```diff
@@ -69,21 +69,32 @@ Feature: Status Desktop community
         When the user creates a community named myCommunity, with description My community description, intro Community Intro and outro Community Outro
         Then the user lands on the community named myCommunity
         When the admin creates a community channel named <community_channel_name>, with description Some description with the method <method>
-        And the admin creates a community category named <community_category_name>, with channel <community_channel_name> and with the method <method>
-        Then the category named <community_category_name> is created
+        And the admin creates a community category named <community_category_name>, with channels <community_channel_name> and with the method <method>
+        Then the category named <community_category_name> has channels <community_channel_name>
 
         Examples:
             | community_channel_name    | community_category_name     | method           |
             | test-channel-1            | test-category-1             | bottom_menu      |
             | test-channel-2            | test-category-2             | right_click_menu |
 
 
+    Scenario: Admin edits a community category
+        When the user creates a community named myCommunity, with description My community description, intro Community Intro and outro Community Outro
+        Then the user lands on the community named myCommunity
+        When the admin creates a community channel named test-channel, with description My description with the method bottom_menu
+        And the admin creates a community category named test-category, with channels test-channel and with the method bottom_menu
```

Comment:
Could `the method bottom_menu` be written more as a user spec / user perspective?

Reply:
Opened issue for that https://github.com/status-im/status-desktop/issues/7105